ktlint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Run ktlint but exclude rules that are currently not applied to this project.
+
+if ./ktlint | grep -Ev "(Wildcard import|Unexpected spacing before)"; then
+  echo "Please correct above lint issues."
+  exit 1
+else
```

Comment:
Nit: Could drop the `else` branch here as you're already `exit`ing above.

Reply:
Technically true, but I usually find it easier to understand the flow if `if` and `else` are clearly marked.